cafebabel/tests/test_articles.py

```diff
@@ -49,7 +49,7 @@ def test_published_article_should_display_content(client, published_article,
     assert published_article.authors[0].profile.name in response
     assert ('href="https://twitter.com/share?url=http%3A%2F%2Flocalhost%2F'
             f'en%2Farticle%2F{published_article.slug}-{published_article.id}'
-            f'%2F&text={published_article.title}&via=cafebabel_eng"'
+            f'%2F&text={published_article.title}&via={twitter_nickname}"'
```

Comment:
Tu veux garder `via=cafebabel_eng` ici

Reply:
yep, I created a new branch from master, I don't understand why I get this here. My bad probably ;-)